src/basicfuns.jl

```diff
@@ -11,14 +11,14 @@ julia> StatsFuns.xlogx(0)
 0.0
 ```
 """
-xlogx(x::Real) = x > zero(x) ? x * log(x) : zero(log(x))
+xlogx(x::Real) = x != zero(x) ? x * log(x) : zero(log(x))
 
 """
     xlogy(x::Real, y::Real)
 
-Return `x * log(y)` for `y > 0` with correct limit at `x = 0`.
+Return `x * log(y)` for `y ≥ 0` with `x == y = 0` evaluating as limit of xlog(x) as x -> 0 from above.
```

Comment:
How about being even more explicit?
```suggestion
Return `x * log(y)` for `y ≥ 0`, or `0` when `x == 0` (consistent with the limit
when ```x \\to 0``` from above, including by extension when `y == 0`).
```

Reply:
sure, looks good.